workspaces/osm/styles/places_dark.css

```diff
@@ -59,7 +59,7 @@
     'Noto Sans Telugu UI Regular'
     'Noto Sans Thaana Regular'
     'Noto Sans Thai UI Regular'
-    'Noto Sans Tibetan Regular'
+    'Noto Sans Tibetan'

```

Comment:
No KR font in this one? Same in places_light, waterareas_dark_labels and waterareas_light_labels, waterways dark and light labels